src/components/structures/RoomDirectory.tsx

```diff
@@ -325,7 +304,7 @@ export default class RoomDirectory extends React.Component<IProps, IState> {
      * HS admins to do this through the RoomSettings interface, but
      * this needs SPEC-417.
      */
-    private removeFromDirectory(room: IRoom) {
+    private removeFromDirectory(room: IPublicRoomsChunk) {
```

Comment:
after seeing this in usage, maybe we should call it `IPublicRoomsRecord` or something, as a chunk implies an array.

Reply:
I think keeping with the spec names of the structures is very valuable

Reply:
The spec names aren't always the greatest. At the very least, this shouldn't imply an array.

Other suggestions:
* `IPublicRoomsChunkRoom`
* `IPublicRoomsChunksChunk`
* `IPublicRoomsChunkRow`
* `IPublicRoomsChunkRecord`